DatadogSessionReplay/SRSnapshotTests/SRFixtures/Sources/SRFixtures/ViewControllers/SwiftUIView.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+struct SwiftUIView: View {
+    var body: some View {
+        VStack(spacing: 10) {
+            Text("Hello, SwiftUI!")
+                .font(.headline)
+
+            if #available(iOS 14.0, *) {
+                if #available(iOS 15.0, *) {
```

Comment:
suggestion/ Since this is only for snapshots the availability in the `SwiftUIView` should be enough with `@available(iOS 15.0, *)`

I'm thinking about the tests starting to fail and we can't understand immediately that one block of the view was available in the new test environment.